pkg/cloud_provider/file/file.go

```diff
@@ -141,10 +155,16 @@ func (manager *gcfsServiceManager) GetInstance(ctx context.Context, obj *Instanc
 	return nil, nil
 }
 
-func cloudInstanceToServiceInstance(instance *beta.Instance) *Instance {
+func cloudInstanceToServiceInstance(instance *beta.Instance) (*Instance, error) {
```

Comment:
I would rename `Instance` to `ServiceInstance` or something to disambiguate from the cloud instance. Currently its hard to tell when you have which one

Reply:
done